packages/curve-ui-kit/src/widgets/Footer/Description.tsx

```diff
@@ -39,7 +39,7 @@ export const Description = () => (
         color="textPrimary"
         sx={{
           fontSize: Sizing[600], // This is a one-off specifically for the logo text
-          fontWeight: 'bold',
+          fontWeight: 'bold !important',
```

Comment:
This doesn't appear to be necessary, `bold` works fine without the `!important`.

Reply:
yeah I changed the wrong field it was supposed to be on the fontSize :fearful: